dxr/plugins/omniglot.py

```diff
@@ -192,12 +192,16 @@ def synth_web_url(self, repo):
         if repo.startswith("git@github.com:"):
             self._is_github = True
             return "https://github.com/" + repo[len("git@github.com:"):]
-        elif repo.startswith("git://github.com/"):
+        elif repo.startswith(("git://github.com/", "https://github.com/")):
             self._is_github = True
             if repo.endswith(".git"):
                 repo = repo[:-len(".git")]
-            return "https" + repo[len("git"):]
-        raise Exception("I don't know what's going on")
+            if repo.startswith("git:"):
+                repo = "https" + repo[len("git"):]
+            return repo
+        else:
```

Comment:
Ok, no problem, I just found that an onboarding with an error saying: "I don't know what's going on" was a bit weird. Maybe we could change it with an error saying something like this: "This remote is not supported for now, please use a remote Github repository. You are welcome to fix it, see bug XXX."